deeprank2/neuralnets/gnn/foutnet.py

```diff
@@ -70,7 +69,18 @@ def __repr__(self):
         return f"{self.__class__.__name__}({self.in_channels}, {self.out_channels})"
 
 
-class FoutNet(nn.Module):  # noqa: D101
+class FoutNet(nn.Module):
+    """FoutNet.
+
+    Architecture based on the FoutLayer. It also uses community pooling to reduce the number of nodes.
+    It can be used for both regression and classification tasks.
```

Comment:
idem above

```suggestion
    """Architecture based on the FoutLayer.

    This architecture uses community pooling to reduce the number of nodes.
    It can be used for both regression and classification tasks.
```